### ISSUE TITLE
scyllatop.py has no -c option

### ISSUE THREAD

**Issue by tgrabiec** [scylladb/scylla#949]
The help string says:
```
use the -c option to give you a configuration example
```

but `-c` is not recognized:

```
scyllatop.py: error: unrecognized arguments: -c
```

**Comment by haarcuba** [comment]
yes, my mistake. It used to be -c, but now it's --print-config. Will fix. Thanks.

**Comment by avikivity** [comment]
Fixed by 651aa06c3278daeb4cd1b729d09b32236592944c
